code/src/Components/Footer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import logo from '../assets/logo.svg';
+
+const Footer = () => {
+  return (
+    <footer>
+      <img alt="Technigo logo" className="logo" src={logo}></img>
+        <div className="footer-info">
+          <p>Project 12 - Movies Project</p>
+          <p>Anna Hellqvist @ Technigo Bootcamp 2020</p>
+          <div>
+            <p>
+              <a href="https://github.com/AnnaHellqvist">GitHub</a>
```

Comment:
The links was a little bit hard to see on my macbook, since you use a black background on your footer. Maybe consider styling them a little bit with a brighter color? :) 🎨

Reply:
changed the color, thanks for the feedback! 😄